[2024-01-19T19:37:48.164-0800] {processor.py:153} INFO - Started process (PID=10642) to work on /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:37:48.164-0800] {processor.py:743} INFO - Processing file /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2024-01-19T19:37:48.165-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:37:48.165-0800] {dagbag.py:532} INFO - Filling up the DagBag from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:37:48.169-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:37:48.167-0800] {dagbag.py:341} ERROR - Failed to import: /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2024-01-19T19:37:48.169-0800] {processor.py:755} WARNING - No viable dags retrieved from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:37:48.202-0800] {processor.py:175} INFO - Processing /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py took 0.042 seconds
[2024-01-19T19:38:33.358-0800] {processor.py:153} INFO - Started process (PID=10708) to work on /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:38:33.381-0800] {processor.py:743} INFO - Processing file /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2024-01-19T19:38:33.382-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:38:33.381-0800] {dagbag.py:532} INFO - Filling up the DagBag from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:38:33.384-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:38:33.383-0800] {dagbag.py:341} ERROR - Failed to import: /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2024-01-19T19:38:33.384-0800] {processor.py:755} WARNING - No viable dags retrieved from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:38:33.406-0800] {processor.py:175} INFO - Processing /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py took 0.053 seconds
[2024-01-19T19:39:18.673-0800] {processor.py:153} INFO - Started process (PID=10759) to work on /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:39:18.675-0800] {processor.py:743} INFO - Processing file /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2024-01-19T19:39:18.676-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:39:18.676-0800] {dagbag.py:532} INFO - Filling up the DagBag from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:39:18.679-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:39:18.678-0800] {dagbag.py:341} ERROR - Failed to import: /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2024-01-19T19:39:18.679-0800] {processor.py:755} WARNING - No viable dags retrieved from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:39:18.705-0800] {processor.py:175} INFO - Processing /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py took 0.036 seconds
[2024-01-19T19:40:07.026-0800] {processor.py:153} INFO - Started process (PID=10817) to work on /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:40:07.030-0800] {processor.py:743} INFO - Processing file /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2024-01-19T19:40:07.031-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:40:07.031-0800] {dagbag.py:532} INFO - Filling up the DagBag from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:40:07.034-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:40:07.033-0800] {dagbag.py:341} ERROR - Failed to import: /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2024-01-19T19:40:07.034-0800] {processor.py:755} WARNING - No viable dags retrieved from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:40:07.063-0800] {processor.py:175} INFO - Processing /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py took 0.040 seconds
[2024-01-19T19:40:51.871-0800] {processor.py:153} INFO - Started process (PID=10859) to work on /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:40:51.873-0800] {processor.py:743} INFO - Processing file /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2024-01-19T19:40:51.873-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:40:51.873-0800] {dagbag.py:532} INFO - Filling up the DagBag from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:40:51.878-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:40:51.875-0800] {dagbag.py:341} ERROR - Failed to import: /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2024-01-19T19:40:51.878-0800] {processor.py:755} WARNING - No viable dags retrieved from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:40:51.904-0800] {processor.py:175} INFO - Processing /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py took 0.037 seconds
[2024-01-19T19:41:43.544-0800] {processor.py:153} INFO - Started process (PID=10903) to work on /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:41:43.548-0800] {processor.py:743} INFO - Processing file /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2024-01-19T19:41:43.550-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:41:43.549-0800] {dagbag.py:532} INFO - Filling up the DagBag from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:41:43.553-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:41:43.552-0800] {dagbag.py:341} ERROR - Failed to import: /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2024-01-19T19:41:43.553-0800] {processor.py:755} WARNING - No viable dags retrieved from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:41:43.583-0800] {processor.py:175} INFO - Processing /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py took 0.042 seconds
[2024-01-19T19:42:42.738-0800] {processor.py:153} INFO - Started process (PID=10952) to work on /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:42:42.761-0800] {processor.py:743} INFO - Processing file /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2024-01-19T19:42:42.763-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:42:42.762-0800] {dagbag.py:532} INFO - Filling up the DagBag from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:42:42.769-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:42:42.768-0800] {dagbag.py:341} ERROR - Failed to import: /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2024-01-19T19:42:42.770-0800] {processor.py:755} WARNING - No viable dags retrieved from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:42:42.827-0800] {processor.py:175} INFO - Processing /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py took 0.096 seconds
[2024-01-19T19:43:40.659-0800] {processor.py:153} INFO - Started process (PID=11002) to work on /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:43:40.661-0800] {processor.py:743} INFO - Processing file /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2024-01-19T19:43:40.662-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:43:40.662-0800] {dagbag.py:532} INFO - Filling up the DagBag from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:43:40.665-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:43:40.664-0800] {dagbag.py:341} ERROR - Failed to import: /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2024-01-19T19:43:40.665-0800] {processor.py:755} WARNING - No viable dags retrieved from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:43:40.695-0800] {processor.py:175} INFO - Processing /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py took 0.042 seconds
[2024-01-19T19:44:24.436-0800] {processor.py:153} INFO - Started process (PID=11040) to work on /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:44:24.438-0800] {processor.py:743} INFO - Processing file /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2024-01-19T19:44:24.439-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:44:24.439-0800] {dagbag.py:532} INFO - Filling up the DagBag from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:44:24.442-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:44:24.441-0800] {dagbag.py:341} ERROR - Failed to import: /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2024-01-19T19:44:24.443-0800] {processor.py:755} WARNING - No viable dags retrieved from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:44:24.563-0800] {processor.py:175} INFO - Processing /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py took 0.132 seconds
[2024-01-19T19:45:09.750-0800] {processor.py:153} INFO - Started process (PID=11082) to work on /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:45:09.753-0800] {processor.py:743} INFO - Processing file /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2024-01-19T19:45:09.754-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:45:09.754-0800] {dagbag.py:532} INFO - Filling up the DagBag from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:45:09.757-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:45:09.757-0800] {dagbag.py:341} ERROR - Failed to import: /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2024-01-19T19:45:09.758-0800] {processor.py:755} WARNING - No viable dags retrieved from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:45:09.787-0800] {processor.py:175} INFO - Processing /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py took 0.043 seconds
[2024-01-19T19:45:57.541-0800] {processor.py:153} INFO - Started process (PID=11116) to work on /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:45:57.543-0800] {processor.py:743} INFO - Processing file /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2024-01-19T19:45:57.544-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:45:57.544-0800] {dagbag.py:532} INFO - Filling up the DagBag from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:45:57.549-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:45:57.546-0800] {dagbag.py:341} ERROR - Failed to import: /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2024-01-19T19:45:57.549-0800] {processor.py:755} WARNING - No viable dags retrieved from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:45:57.586-0800] {processor.py:175} INFO - Processing /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py took 0.051 seconds
[2024-01-19T19:46:44.132-0800] {processor.py:153} INFO - Started process (PID=11158) to work on /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:46:44.176-0800] {processor.py:743} INFO - Processing file /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2024-01-19T19:46:44.177-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:46:44.177-0800] {dagbag.py:532} INFO - Filling up the DagBag from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:46:44.179-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:46:44.179-0800] {dagbag.py:341} ERROR - Failed to import: /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2024-01-19T19:46:44.180-0800] {processor.py:755} WARNING - No viable dags retrieved from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:46:44.208-0800] {processor.py:175} INFO - Processing /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py took 0.080 seconds
[2024-01-19T19:47:33.129-0800] {processor.py:153} INFO - Started process (PID=11204) to work on /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:47:33.130-0800] {processor.py:743} INFO - Processing file /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2024-01-19T19:47:33.131-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:47:33.131-0800] {dagbag.py:532} INFO - Filling up the DagBag from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:47:33.134-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:47:33.133-0800] {dagbag.py:341} ERROR - Failed to import: /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2024-01-19T19:47:33.135-0800] {processor.py:755} WARNING - No viable dags retrieved from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:47:33.166-0800] {processor.py:175} INFO - Processing /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py took 0.041 seconds
[2024-01-19T19:48:23.403-0800] {processor.py:153} INFO - Started process (PID=11247) to work on /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:48:23.406-0800] {processor.py:743} INFO - Processing file /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py for tasks to queue
[2024-01-19T19:48:23.408-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:48:23.408-0800] {dagbag.py:532} INFO - Filling up the DagBag from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:48:23.413-0800] {logging_mixin.py:137} INFO - [2024-01-19T19:48:23.412-0800] {dagbag.py:341} ERROR - Failed to import: /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
Traceback (most recent call last):
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py", line 24, in <module>
    from pandas.tseries.holiday import USFederalHolidayCalendar
ModuleNotFoundError: No module named 'pandas'
[2024-01-19T19:48:23.413-0800] {processor.py:755} WARNING - No viable dags retrieved from /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py
[2024-01-19T19:48:23.458-0800] {processor.py:175} INFO - Processing /home/mirukulla/.local/lib/python3.10/site-packages/airflow/example_dags/plugins/workday.py took 0.063 seconds
